### PR TITLE
show toast if any chk request failed

### PR DESCRIPTION
**Pull request by jgfidelis**
#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -65,13 +70,14 @@ class MiniCart extends Component {
...
   handleItemsDifference = async clientItems => {
+    const { showToast, intl } = this.props
     this.setState({ updatingOrderForm: true })
     try {
       const items = clientItems.map(
         pick(['id', 'index', 'quantity', 'seller', 'options'])
       )
-      const addItemsResponse = await this.addItems(items)
       const updateItemsResponse = await this.updateItems(items)
+      const addItemsResponse = await this.addItems(items)
```
Comment (jgfidelis) [inline react/index.js:80]: Troquei elas de ordem pq causava um bugzinho na ordem antiga

File: messages/pt.json [hunk 2]
```diff
@@ -13,5 +13,6 @@
     "sidebar-title": "Meu carrinho",
     "minicart-content-footer-discount": "Economize",
     "minicart.error-removal": "A remoção do item falhou, por favor tente novamente",
-    "minicart.shipping-cost": "Custo de entrega"
+    "minicart.shipping-cost": "Custo de entrega",
+    "minicart.checkout-failure": "Houve um problema com sua requisição. Por favor, tente novamente."
```
Comment (klzns) [inline messages/pt.json]: I would remove technical terms like "requests", "requisição", etc.

File: react/index.js [hunk 3]
```diff
@@ -35,7 +36,11 @@ const DEFAULT_ICON_CLASSES = 'gray'
  * Minicart component
  */
 class MiniCart extends Component {
-  static propTypes = MiniCartPropTypes
+  static propTypes = { 
+    ...MiniCartPropTypes,
+    intl: intlShape,
```
Comment (lbebber) [inline react/index.js]:
```suggestion
    intl: intlShape.isRequired,
```

**Review by klzns — APPROVED** [review]

**Review by lbebber — APPROVED** [review]
1 suggestion + changelog

**Pull request merged**